mllib/src/main/scala/org/apache/spark/mllib/tree/loss/AbsoluteError.scala

```diff
@@ -61,4 +61,18 @@ object AbsoluteError extends Loss {
       math.abs(err)
     }.mean()
   }
+
+  /**
```

Comment:
I think it's OK for the doc for gradient() and computeError() to be generic as long as the doc for the loss classes describes the specific loss function.